### ISSUE TITLE
Inconsistent behaviour with inventory items

### ISSUE THREAD

**Issue by wrincewind** [FTBTeam/FTB-Chunks#121]
https://github.com/Vazkii/Patchouli/issues/345 

TL;DR:

some items in player inventories open UIs when right clicked. Some of these can't be accessed when standing in another team's claimed chunk if you aren't a member. E.G. you can open a vanilla *Book and Quill*, but as soon as you sign it, you can no longer open it. Some modded items can be opened (e.g. the astral sorcery tome), but others can't (e.g. patchouli guidebooks) and I can't see any rhyme or reason as to which items work and which ones don't.

I assume that some of them are calling methods that're blocked by chunk claims and others aren't, but as to what those methods are, I've no idea.

**Comment by MaxNeedsSnacks** [comment]
Should be fixed in the newest release, right-click actions are now much, much more permissive
